app/templates/components/sl-split-grid.hbs

```diff
@@ -0,0 +1,111 @@
+<header class="grid-header">
+    <div class="row">
+        <div class="col-sm-6">
+            {{#if view.headerTemplate}}
+                <h3 class="title">
+                    {{partial view.headerTemplate}}
+                </h3>
+            {{/if}}
+        </div>
+
+        {{#if filterName}}
+            <div class="col-sm-6 text-right">
+                {{#sl-button action="toggleFilterPane" size="small"}}{{filterText}}{{/sl-button}}
```

Comment:
I'm not sure. The spec specifies that the related container, ".filter-pane" in this case, should be referenced here as aria-controls, but that means passing around another ID. We can do that, but I'm not clear on whether this exact use is really part of the intended scope of the property.